### DISCUSSION TITLE
Work Time

### DISCUSSION THREAD

**Discussion by harshit-paneri** [Sulagna-Dutta-Roy/Webeginner-projects#95] [General]
What is Ideal Time of an Assign Issue.

**Comment by Sulagna-Dutta-Roy** [comment]
There's no time for project admins for assign an Issue. Because admins have others work also mentors have the same. Open source is all about patience and learning don't worry we'll try our best also assign PRs etc.

**Reply by harshit-paneri** [reply]
Okii, Sure.
